TriblerGUI/widgets/trustpage/js/data_processor.js

```diff
@@ -173,6 +178,34 @@ function addMinMaxTransmission(response, interim) {
 }
 
 /**
+ * Calculate the base and the slope for the linear function describing the amount of traffic.
+ *
+ * @param {GraphResponseData} response - The response
+ * @param {Object} interim - The interim result
+ * @returns {{traffic_slope: number, traffic_base: number}}
+ */
+function addTrafficFunction(response, interim) {
+    var min = Number.MAX_VALUE,
+        max = Number.MIN_VALUE;
+
+    interim.nodes.forEach(function (node) {
+       var sum = node.total_up + node.total_down;
+       if (sum < min) min = sum;
+       if (sum > max) max = sum;
+    });
+
+    min = min === Number.MAX_VALUE ? 0 : min;
+    max = max === Number.MIN_VALUE ? 1 : max;
+
+    var slope = (config.node.circle.maxRadius - config.node.circle.minRadius) / (max - min);
```

Comment:
please keep config values out of this file. This can be done by moving the config part to drawing.js. This way we separate drawing logic from data logic.